Makefile

```diff
@@ -1,25 +1,37 @@
-TARGET        = phpbrew
-SIGNATURE     = $(TARGET).asc
+# See https://tech.davis-hansson.com/p/make/
+MAKEFLAGS += --warn-undefined-variables
+MAKEFLAGS += --no-builtin-rules
+
+PHPBREW_PHAR        = phpbrew
+SIGNATURE     = $(PHPBREW_PHAR).asc
 CP            = cp
 INSTALL_PATH  = /usr/local/bin
 TEST          = phpunit
 
-$(TARGET): vendor $(shell find bin/ shell/ src/ -type f) box.json.dist .git/HEAD
-	box compile
-	touch -c $@
+PHAR_SRC_FILES := $(shell find bin/ shell/ src/ -type f)
+
+
+.DEFAULT_GOAL := help
```

Comment:
Introduce a help command. I don't think it is wise to display all the commands there, but it will be good to document the most important ones. For now I've only documented the building PHAR one.